connect_ext_ppr/tasks_manager.py

```diff
@@ -82,6 +90,50 @@ def _check_cbc_task_status(cbc_service, tracking_id):
     raise TaskException(f'Something went wrong with task: {tracking_id}')
 
 
+def prepare_ppr_file_for_task(
+    connect_client,
+    file_data,
+    deployment_request,
+    deployment,
+    process_func,
+):
+    file, writer, wb = get_base_workbook(file_data)
+
+    try:
+        ws_list = []
+        for sheet_name in wb.sheet_names:
+            ws = wb.parse(sheet_name)
+            process_func(sheet_name, ws)
+            ws.name = sheet_name
+            ws_list.append(ws)
+
+        for ws in ws_list:
+            ws.to_excel(writer, ws.name, index=False)
+        file_obj = open(file.name, 'rb')
+        writer.book.save(file_obj.name)
+
+        file_size = get_file_size(file_obj)
+        file_name = PPR_FILE_NAME_DELEGATION_L2.format(
```

Comment:
[COD-C] if we are going to reuse this method for the other task, we should ask the name or name template as argument.